README.md

```diff
@@ -150,6 +150,17 @@ being:
 - `i + 1` replaces `1,1,1,1,1` into `1,2,3,4,5`, assuming that each selection is
   on a different digit.
 
+### Status bar
+
+Dance provides several status bar segments (left-aligned) exposing info similar
+to Kakoune's default mode-line:
```

Comment:
I would also add that most of them are hidden by default, and only shown contextually. You can either write this, or explain it individually for each segment.